examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>This plot is really cool. I have a couple of suggestions:</p><ol><li>I wouldn't share the y-axis or clip it so that the distributions can be better seen</li><li>Maybe do clip the x-axis on the right (45 or something)</li><li>Show the true fraction (with a dashed vertical line or something not very overwhelming). This way you clearly show not only the bias in variance but also the bias in location.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<blockquote>I wouldn't share the y-axis or clip it so that the distributions can be better seen</blockquote><p> </p><p>Yeah, I get that. I <em>do</em> want them to be comparable across indexes, though, so maybe I could clip all of them (including the fifth) to 0.5?</p><p> </p><p>Or clip the first four there, and keep the last one at the full span?</p><p> </p><blockquote>Maybe do clip the x-axis on the right (45 or something)</blockquote><p> </p><p>I prefer the full span. It does mean there's more empty space, but I think it better communicates what the domain of the (implicit) mass function is.</p><p> </p><blockquote>Show the true fraction (with a dashed vertical line or something not very overwhelming). This way you clearly show not only the bias in variance but also the bias in location.</blockquote><p> </p><p>Great idea!</p>

Reply:
<p>Super nice @bsmith89! Do you think you could also add the species in the legend (or elsewhere) for each plot? That way we know which color is what ;)</p><p> </p>

Reply:
<p>Labels added as legend titles.</p>